src/lib.rs

```diff
@@ -164,11 +159,10 @@ pub async fn start_lemmy_server() -> Result<(), LemmyError> {
 
   // this must come before the HttpServer creation
   // creates a middleware that populates http metrics for each path, method, and status code
-  #[cfg(feature = "prometheus-metrics")]
   let prom_api_metrics = PrometheusMetricsBuilder::new("lemmy_api")
     .registry(default_registry().clone())
     .build()
-    .expect("Should always be buildable");
+    .expect("Failed to initialize actix_web_prom metrics");
```

Comment:
We can move this inside prometheus_metrics.rs to shorten this file a bit.

Reply:
Specifically, return PrometheusMetrics from `serve_prometheus_metrics`. Then you will have an `Option<PrometheusMetrics>`. Use `if let Some(metrics) = metrics { app.wrap(metrics) }`